wcivf/apps/people/templates/people/not_current_person_detail.html

```diff
@@ -0,0 +1,79 @@
+{% extends "base.html" %}
+{% load markdown_deux_tags %}
+{% load breadcrumb_tags %}
+{% load humanize %}
+
+{% block page_title %}{% include "people/includes/_person_meta_title.html" %}{% endblock page_title %}
+{% block page_description %}{% include "people/includes/_person_meta_description.html" %}{% endblock page_description %}
+{% block og_title %}{% include "people/includes/_person_meta_title.html" %}{% endblock og_title %}
+{% block og_description %}{% include "people/includes/_person_meta_description.html" %}{% endblock og_description %}
+{% block og_image %}{% if object.photo_url %}{{ object.photo_url }}{% endif %}{% endblock og_image %}
+
+{% block page_meta %}<meta name="robots" content="noindex">{% endblock page_meta %}
+
+{% block content %}
+  <div class="card person_banner">
+    <h2 class="person_name">{{ object.name }}
+    {% if person.death_date and person.current_or_future_candidacies and person.personpost.party.is_independent %}
+      (Deceased)
+    {% endif %}
+    </h2>
+    {% if object.photo_url %}
+      <img src="{{ object.photo_url }}" class="person_photo detail" alt="profile photo of {{ object.name }}">
+    {% endif %}
+
+    <p>
+    {{ object.intro|safe }}. Our volunteers have been adding information on {{ object.name }} - here's everything we know so far!
```

Comment:
"Our volunteers have been adding information on {{ object.name }} - here's everything we know so far!" can go from this line